src/plugins/data/public/data_sources/constants.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { DataSourceUIGroupType } from './datasource/types';
+import { DEFAULT_DATA_SOURCE_DISPLAY_NAME } from './register_default_datasource';
+
+export const defaultDataSourceMetadata = {
```

Comment:
qq: did we settle on this code wise being internally referenced as data source?

Reply:
In last Thursday's meeting @kgcreative mentioned we should for now only aligns on data connector in replace of previous data source selector naming, but not for renaming other existing datasource names through out the code for this. Soon will have a push for renaming stuff only for datasource selector to be data selector.